src/app/models/product.ts

```diff
@@ -7,12 +7,12 @@ export class Product {
   public name: string;
   public category: string;
   public description: string;
-  public heroImageUrl: string;
-  public imageUrls:string[]=[];
+  public heroImage: string;
+  public images:string[]=[];
   public price=new Price();
   public pickupAddress = new Address();
   public status: string;
-  public postDate: Date;
+  public postDateTime: string;
```

Comment:
because when we are receiving response, the date is getting converted to sting format. To handle that situation we are keeping this as string in model and then converting later while using it.